packages/bierzo-wallet/src/logic/codec.unit.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { ChainId } from '@iov/bcp';
+import { EthereumCodec } from '@iov/ethereum';
+
+import { withChainsDescribe } from '../utils/test/testExecutor';
+import { getCodecForChainId } from './codec';
+import { disconnect } from './connection';
+
+withChainsDescribe('Logic :: codec', () => {
+  afterAll(async () => {
+    await disconnect();
```

Comment:
This `disconnect()` was changed to synchronous. So no `async` and `await` needed